packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/creditCard.gql.js

```diff
@@ -5,19 +5,6 @@ import gql from 'graphql-tag';
 // https://github.com/apollographql/eslint-plugin-graphql/issues/99
 /* eslint-disable graphql/template-strings */
 
-export const GET_ALL_COUNTRIES = gql`
```

Comment:
Why did you remove this query? We were previously validating the countries but that has now changed to just checking that the field is there, not that it's valid. If the change to validation was necessary (I'm not sure it is as we can just hide the form right?) then lets try to find a way to include country validation so we don't lose this functionality.

Reply:
We were using this query to get the list of countries. The list of countries are being used to validate the region code. But the region code validator that we have only works for US. It does not work for other countries. Hence I am doing what Shipping Address form does, only validate if it is present or not.

This is not the right place for that. A robust validate function needs to be written into the new Region component to handle such cases. It will be handled in a new ticket.